client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/design-picker-design-title.tsx

```diff
@@ -29,11 +30,21 @@ const DesignPickerDesignTitle: FC< Props > = ( { designTitle, selectedDesign } )
 		)
 	);
 
-	const showBundledBadge = selectedDesign.is_bundled_with_woo_commerce;
+	const bundleSettings = useBundleSettings( selectedDesign.slug );
```

Comment:
If I understand correctly, the `selectedDesign` always will be a theme, so `selectedDesign.slug` is the same thing as `themeId` from other parts of the application. Is it correct?